packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -329,6 +329,7 @@ export async function startTransformationJob(uploadId: string) {
 
     await sleep(2000) // sleep before polling job to prevent ThrottlingException
     throwIfCancelled()
+    sessionPlanProgress['uploadCode'] = StepProgress.Succeeded
```

Comment:
Question: What is this line doing/changing?

Reply:
Oh this is used to control when the green check mark appears next to each overall step (i.e. "Waiting for job to start", etc.) -- I'll rename this to 'startJob' instead of 'uploadCode' to be more clear -- here I set it to Succeeded right after the startJob API call was successful, so that next to "Waiting for job to start" we can show a green check mark.